ckanext/geocat/utils/csw_mapping.py

```diff
@@ -78,12 +81,17 @@ def get_metadata(self, csw_record_as_string, geocat_id):
         dataset_dict['owner_org'] = self.organization_slug
         rights = \
             _map_dataset_rights(node=root_node,
-                                terms_of_use=self.terms_of_use_graph)
+                                terms_of_use=self.terms_of_use_graph,
+                                default_rights=self.default_rights)
 
         dataset_dict['relations'] = []
         dataset_dict['resources'] = []
         download_formats = _get_download_distribution_formats(node=root_node)
         service_formats = _get_service_distribution_formats(node=root_node)
+        log.debug("The following service formats are detected: {}"
+                  .format(service_formats))
+        log.debug("The following download formats are detected: {}"
+                  .format(download_formats))
```

Comment:
I kind of felt it might be good to log the formats, since the way we map them is very messy: we split them out of the protocol name. Initially I wanted to validate the format we get against those formats specified, but somehow I dropped this approach because that way not all formats were mapped as they were expected to be mapped.